docs/howto/ios-tips.md

```diff
@@ -84,3 +84,87 @@ It seems like there's some caching strategy to avoid fetching
 `.podspec` files unnecessarily, potentially with network requests.
 (See
 [discussion](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/.23M3548.20RN.20v0.2E60.2E0.20upgrade/near/896746).)
+
+## Sign in with Apple
+
+To set up your [development server](./dev-server.md) to use Apple
+authentication ("Sign in with Apple"), you'll want to follow almost
+[these
+steps](https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#sign-in-with-apple),
+but with a few things to keep in mind:
+
+- If you don't have your own Apple Developer account (there's an
+  annual fee), please ask Greg to set up test credentials and send
+  them to you.
+  These will be associated with the Kandra team, so
+  [please](https://chat.zulip.org/#narrow/stream/3-backend/topic/apple.20auth/near/915391)
+  let him know when you're finished with the credentials so he can
+  revoke them. Please don't abuse them with deliberate spam, as
+  that goes on our reputation.
+- Use the domain `zulipdev.com` where Apple asks for a domain;
+  [`localhost` won't
+  work](https://chat.zulip.org/#narrow/stream/3-backend/topic/Apple.20Auth/near/831533).
+  - Add a line like `127.0.0.1 zulipdev.com` to your `/etc/hosts` as
+    described [here](https://zulip.readthedocs.io/en/latest/subsystems/realms.html?highlight=zulipdev.com#working-with-subdomains-in-development-environment). Notice
+    there's already a line mapping 127.0.0.1 to `localhost`; this is what
+    makes the iOS simulator work with `http://localhost:9991`, as
+    described [here](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/dev-server.md#ios-simulator-macos-only), without
+    having to configure `EXTERNAL_HOST`. Likewise, you won't have to
+    set `EXTERNAL_HOST` to get it to work on the simulator.
+  - A physical device won't be aware of the `zulipdev.com` domain;
```

Comment:
Because the domain resolves on the public Internet to the right address, the physical device will be aware of the domain; the trouble is that it'll mean the wrong thing. 127.0.0.1 is a [loopback](https://en.wikipedia.org/wiki/Localhost) address, pointing at the current machine you're on -- so when you're on a physical device, that's the device itself. (Arguably when you're in the iOS simulator, it ought to mean the simulated device, but empirically it doesn't.)